### DISCUSSION TITLE
How to check  detail Information about Edge Function ?

### DISCUSSION THREAD

**Discussion by coleea** [vercel/community#2427] [Help] [answered]
### Summary

now my web-app is running on Edge runtime by inserting `export const runtime = 'experimental-edge'; ` on my server copmonent
I checked that this edge runtime works correctly.
but I want to know Information about Edge Function such as

* runtime information
* location in which the edge function is called
* execution duration
* cold boot duration
* memory used

I want these information for comparing between edge function and Serverless function

As you can see from the screenshot below, I managed to find some information but It is not enough

thank you

<img width="320" alt="____제목 없음" src="https://github.com/vercel/community/assets/7471674/0296c374-55fc-475b-b156-728a132d54e8">

**Comment by amyegan** [comment, marked as answer]
Hi @colea! You can add [Monitoring](https://vercel.com/docs/concepts/observability/monitoring/overview#) to a Pro plan or use an external monitoring tool to collect more detailed data about your Serverless and Edge Functions. We have a few [monitoring integration options](https://vercel.com/integrations#monitoring) to make the process easier, but you not limited to that list if there is another tool that you prefer.

**Reply by coleea** [reply]
thank you for answer 😀
